kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KopServerStats.java

```diff
@@ -51,6 +51,7 @@ public interface KopServerStats {
     /**
      * PRODUCE STATS.
      */
+    String PENDING_TOPIC_LATENCY = "PENDING_TOPIC_LATENCY";
```

Comment:
We also need to add the new metrics to `docs/reference-metrics.md`.

Reply:
Thanks! Somehow missed it in my search.